src/SDK.ts

```diff
@@ -153,20 +165,22 @@ export default class SDK implements ISDK {
 
     // updated auth endpoint for authenticated chats and add auth token in header
     if (authenticatedUserToken) {
-      endpoint = `${this.omnichannelConfiguration.orgUrl}/${OmnichannelEndpoints.LiveChatAuthLiveWorkItemDetailsPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
+      requestPath = `${this.omnichannelConfiguration.orgUrl}/${OmnichannelEndpoints.LiveChatAuthLiveWorkItemDetailsPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
```

Comment:
`requestPath` should not have `${this.omnichannelConfiguration.orgUrl}`

Reply:
Fixed